central_v2/src/pages/Login.tsx

```diff
@@ -4,7 +4,7 @@ import { styled } from "@mui/system";
 import { Link } from "react-router-dom";
```

Comment:
central_v2 will be reworked in a subsequent set of PRs in Q3 (Design is still working on it)